sys/include/net/ieee802154/radio.h

```diff
@@ -190,6 +194,26 @@ typedef enum {
      */
     IEEE802154_RADIO_INDICATION_RX_START,
 
+    /**
+     * @brief the transceiver received a frame with an invalid crc
+     *        and lies in the internal framebuffer.
+     *
+     * This indication should be generated only if the CRC is invalid and
```

Comment:
If the CRC is wrong the content of the framebuffer is assumed to be wrong as well. I know some radios generate the "AMI" interrupt before checking the CRC, but should we take into consideration the address considering the frame is wrong anyway?

Reply:
I think that makes sense.

Reply:
Then should I re-phrase the brief to:

```
@ brief  the transceiver received a frame with an invalid crc, the frame buffer content is considered invalid.
```